docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -16,35 +16,37 @@ Which browser is used depends on:
 
 The following table shows which browser is used for the various platforms and operating systems.
 
-|**OS / Platform**|**Browser**|
+|**OS**|**Office version**|**Edge WebView2 (Chromium-based) installed**|**Browser**|
 |:-----|:-----|:-----|:-----|:-----|:-----|:-----|
-|Office on the web|The browser in which Office is opened.|
-|Mac|Safari|
-|iOS|Safari|
-|Android|Chrome|
-|Windows / non-subscription Office 2013 or later|Internet Explorer 11|
-|Windows 10 ver. < 1903 / Office 365|Internet Explorer 11|
-|Windows 10 ver. >= 1903 / Office 365 ver < 16.0.11629<sup>1</sup>|Internet Explorer 11|
-|Windows 10 ver. >= 1903 / Office 365 ver >= 16.0.11629<sup>1</sup>|Microsoft Edge<sup>2, 3</sup>|
+|any|Office on the web|N/A|The browser in which Office is opened.|
+|Mac|any|N/A|Safari|
+|iOS|any|N/A|Safari|
+|Android|any|N/A|Chrome|
+|Windows 7, 8.1, 10 | non-subscription Office 2013 or later|Doesn't matter|Internet Explorer 11|
+|Windows 7 | Microsoft 365| Doesn't matter | Internet Explorer 11|
+|Windows 8.1,<br>Windows 10 ver.&nbsp;<&nbsp;1903| Microsoft 365 | No| Internet Explorer 11|
+|Windows 10 ver.&nbsp;>=&nbsp;1903 | Microsoft 365 ver.&nbsp;<&nbsp;16.0.11629<sup>1</sup>| Doesn't matter|Internet Explorer 11|
+|Windows 10 ver.&nbsp;>=&nbsp;1903 | Microsoft 365 ver.&nbsp;>=&nbsp;16.0.11629&nbsp;_AND_&nbsp;<&nbsp;16.0.13127.20082<sup>1</sup>| Doesn't matter|Microsoft Edge<sup>2, 3</sup> with original WebView (EdgeHTML)|
+|Windows 10 ver.&nbsp;>=&nbsp;1903 | Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| No |Microsoft Edge<sup>2, 3</sup> with original WebView (EdgeHTML)|
+|Windows 8.1<br>Windows 10| Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| Yes|  SEE NOTE 4 BELOW.|
```

Comment:
```suggestion
|Windows 8.1<br>Windows 10| Microsoft 365 ver.&nbsp;>=&nbsp;16.0.13127.20082<sup>1</sup>| Yes|  Microsoft Edge<sup>4</sup>|
```

Reply:
I prefer to force the reader to read the note to get any information at all.

Reply:
Ok, updated suggestion